docs/tre-workspace-authors/firewall-rules.md

```diff
@@ -1,128 +1,256 @@
-# Adding Firewall Rules as part of a workspace or service deployment
+# Configuring Firewall Rules in Azure TRE
 
-!!! note
-    Creating firewall rules will in the future only be allowed through the firewall shared services [#882](https://github.com/microsoft/AzureTRE/issues/882).
+This guide explains how to configure firewall rules for your Azure TRE environment.
 
-A TRE service may require certain firewall rules to be opened in the TRE firewall. Examples include:
+## Contents
+- [Overview](#overview)
+- [Types of Firewall Rules](#types-of-firewall-rules)
+- [Configuring Application Rules](#configuring-application-rules)
+- [Configuring Network Rules](#configuring-network-rules)
+- [Applying Your Configuration](#applying-your-configuration)
+- [Examples](#examples)
 
-- Access to an external authorisation endpoint
-- Access to an external data store
-- Access to an external API
+## Overview
 
-Please be aware when opening firewall rules there is the potential for data to be leaked from the workspace to the external location.
+The Azure TRE Firewall shared service controls network traffic flowing in and out of your research environment. The firewall configuration is defined using a JSON file that specifies rule collections and their associated rules.
```

Comment:
We seem to have lost the warning from the original document that I think we should keep. 

Also think  need to describe how it relates to `tempalte_schema.json` and workspaces/services.